openmetadata-ui/src/main/resources/ui/src/pages/UserPage/UserPage.component.tsx

```diff
@@ -271,6 +351,21 @@ const UserPage = () => {
     setEntityThread([]);
   }, [tab]);
 
+  useEffect(() => {
+    if (!isEmpty(userData) && !isUndefined(userData)) {
```

Comment:
isEmpty will also check for undefined one

Reply:
Done